app/helpers/layouts_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module LayoutsHelper
```

Comment:
It will allow to have nested layouts.
I needed to have pages in `exchanges`, but I didn't want to repeat some html structure that the original layout did not have. 
And also I didn't want to create a new layout.